js/tenants.js

```diff
@@ -64,28 +64,48 @@ function saveActiveTenant(tenantCode, cb) {
                     cb();
                 }
             });
-            
+
         }
     });
 }
 
 /**
- * 
+ *
  * @callback allTenants
- * 
- * @param {Object} tenants An object of tenants, keyed by tenant code, with the name as values 
+ *
+ * @param {Object} tenants An object of tenants, keyed by tenant code, with the name as values
  */
 /**
  * Returns all defined tenants
- * 
- * @param {allTenants} cb 
+ *
+ * @param {allTenants} cb
  */
 function getTenants(cb) {
     var tenants = {};
-    for (var code in allTenants) {
-        if (allTenants[code].hasOwnProperty('apps') && allTenants[code].apps.hasOwnProperty('rl')) {
-            tenants[code] = allTenants[code].name;
+    getTenantList(function(tenantList) {
+        for (var code in tenantList) {
+            if (tenantList[code].hasOwnProperty('apps') && tenantList[code].apps.hasOwnProperty('rl')) {
+                tenants[code] = tenantList[code].name;
+            }
         }
-    }
-    cb(tenants);
+        cb(tenants);
+    });
+}
+
+/**
+ * Retrieves the current tenant list or displays a canned list, if error
+ * @param {*} cb
+ */
+function getTenantList(cb) {
+    var headers = new Headers({'Content-Type': 'application/json'});
+    fetch('https://s3-eu-west-1.amazonaws.com/talis-public/talis.com/customers.json', {headers: headers})
```

Comment:
Did i make it up - or was there a discussion about serving this from a talis domain rather than an s3 one?